.gitmodules

```diff
@@ -1,6 +1,6 @@
 [submodule "lib/maplibre-native-bindings-jni/maplibre-native"]
 	path = lib/maplibre-native-bindings-jni/vendor/maplibre-native
-	url = https://github.com/maplibre/maplibre-native.git
+	url = https://github.com/sargunv/maplibre-native.git
```

Comment:
referencing the fork until https://github.com/maplibre/maplibre-native/pull/3791 merges